### PR TITLE
Convert script to python package.

### PR DESCRIPTION
**Pull request by sophiekovalevsky**
I have added a couple of enhancements in how we call kicad jlc tool. 

Here is the summary: 
- Now you can install the application as a python package which will definitely make it more pythonic.
- You can install the application using pip
- Once package is installed in your system you can call it using `kicad-jlc-tool`

You can test it issuing: 
```shell
pip install git+https://github.com/sophiekovalevsky/JLCKicadTools.git@518ec7dac5b1df354ee77905b6233245711f822c
```
Once the merge is done, in the README there is a content that stays how user should download package from upstream repo. 

I am looking forward to your feedback.

### CODE REVIEW

File: kicad_jlc_tool/generate_jlc_files.py [hunk 1]
```diff
@@ -23,8 +23,8 @@
 import logging
 import errno
 
-from jlc_lib.cpl_fix_rotations import ReadDB, FixRotations
-from jlc_lib.generate_bom import GenerateBOM
+from kicad_jlc_tool.jlc_lib.cpl_fix_rotations import ReadDB, FixRotations
```
Comment (matthewlai) [inline kicad_jlc_tool/generate_jlc_files.py]: Can we stick to jlc_kicad_tools? Just so all the names match (repository name, library name, package name, etc).

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+from setuptools import setup, find_packages
+from kicad_jlc_tool import __version__
+
+setup(
+    name='kicad-jlc-tool',
```
Comment (matthewlai) [inline setup.py]: jlc-kicad-tools

**Comment by sophiekovalevsky** [comment]
@matthewlai 

I have made changes you suggested. Let me know if I miss something else.

If there is an additional change I will probably will work on that in about two weeks since I will away from development.

Best,

**Comment by matthewlai** [comment]
Looks good to me. Thanks for working on this!

**Comment by matthewlai** [comment]
Hi @sophiekovalevsky , do you have a moment to take a look at this please?

https://github.com/matthewlai/JLCKicadTools/issues/15

Thanks!

**Comment by sophiekovalevsky** [comment]
@matthewlai I will

**Pull request merged**